website_crm_quick_answer/data/automation.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<data noupdate="1">
+
+<record id="website_form_leads_filter" model="ir.filters">
+    <field name="name">Coming from the website contact form</field>
+    <field name="user_id" eval="False"/>
+    <field name="model_id">crm.lead</field>
+    <field name="domain"
+           eval="[('medium_id', '=', ref('utm.utm_medium_website'))]"/>
```

Comment:
This is noupdate, so migrations will not get this filter correctly and will not get the addon working, right? I guess you'd have to add a migration script of some kind (not sure if here or in OpenUpgrade)